### PR TITLE
[dev]sorted pics to the top

### PR DESCRIPTION
**Pull request by MissyM2**
#### Change
made edits to *about.html* to sort the members with sketches to the top of the page.

#### Testing

- no new errors on the console
- looks visually correct
Screen Shot 2020-10-22 at 1 22 14 PM

### CODE REVIEW

File: _layouts/about.html [hunk 1]
```diff
@@ -32,20 +32,41 @@ <h1 class="hero-header title"> {{ page.header }}</h1>
...
+              <h4 class='name'>{{member.full_name}}</h4>
+              {% if member.role %}<h5 class='role'>{{member.role}}</h5>{% endif %}
+            </div>
+          </a>
+        </li>
+        {% endif %}
         {% endfor %}
       </ul>
     </div>
+    <ul id="members">
```
[the remark this answers is not in the document]
Comment (meissadia) [reply _layouts/about.html:54]: Also, element IDs are supposed to be unique, so can you please switch from `<ul id="members">` to `<ul class="members">` for both occurrences and update the CSS accordingly?

**Review by meissadia — CHANGES_REQUESTED** [review]

File: _layouts/about.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (MissyM2) [reply _layouts/about.html]: I made your updates and I added 'margin-bottom:0' on the ul to get rid of the little bit of extra space.

[the remark this answers is not in the document]
Comment (meissadia) [reply _layouts/about.html]: @MissyM2 I usually resolve the convo once it no longer requires a response.

**Review by meissadia — APPROVED** [review]
🚀

**Pull request merged**